pykafka/rdkafka/simple_consumer.py

```diff
@@ -0,0 +1,152 @@
+from pykafka.simpleconsumer import (
+    SimpleConsumer, ConsumerStoppedException, OffsetType)
+from . import _rd_kafka
+
+
+class RdKafkaSimpleConsumer(SimpleConsumer):
+    """A pykafka.SimpleConsumer with librdkafka-based fetchers
+
+    This aims to conform to the SimpleConsumer interface as closely as
+    possible.  There are some notable differences:
+
+    1. rotating over partitions: while message ordering within partitions is
+       conserved (of course!), the order in which partitions are visited will
+       deviate.  In particular, here we may emit more than one message from
+       the same partition before visiting another
+    2. ignores num_consumer_fetchers: librdkafka will typically spawn at least
+       as many threads as there are kafka cluster nodes
+
+    For an overview of how configuration keys are mapped to librdkafka's, see
+    _mk_rdkafka_config_lists.
+    """
+
+    def _setup_fetch_workers(self):
```

Comment:
Whoever said that thing about variable naming being hard was right... I still haven't thought of a better name. In fact, given that we delegate absolutely everything except message-fetching to `SimpleConsumer`, you could even argue that `self._rdk_consumer` is really just a "fetch worker" for us :)